lib/frontier/views/index/empty_message_and_call_to_action.rb

```diff
@@ -0,0 +1,33 @@
+class Frontier::Views::Index::EmptyMessageAndCallToAction
+
+  include Frontier::ModelConfigurationProperty
+
+  def to_s
+    if model_configuration.show_create?
+      empty_message_with_call_to_action
+    else
+      "%p #{empty_message_without_call_to_action}"
+    end
+  end
+
+private
+
+  def empty_message_with_call_to_action
+    raw = <<-STRING
+%p
+  #{empty_message_without_call_to_action}
+  - if #{Frontier::Authorization::Check.new(model_configuration, model_configuration.as_constant, :new)}
+    = link_to(\"Click here to create one.\", #{model_configuration.url_builder.new_path})
```

Comment:
Hence [my suggestion](https://github.com/thefrontiergroup/frontier_generators/pull/106#issuecomment-196101745) to drop the link from the paragraph and use the main call to action button underneath it instead.